src/CSS.php

```diff
@@ -252,7 +252,7 @@ protected function combineImports($source, $content, $parents)
      */
     protected function importFiles($source, $content)
     {
-        $regex = '/url\((["\']?)(.+?)\\1\)/i';
+	    $regex = '/url\((["\']?)([^"\']+?)\\1\)/i';
```

Comment:
This will be a problem when a quote is encountered inside the url.
I think changing the regex to `'/url\((["\']?)(.*?)\\1\)/i';` instead of `'/url\((["\']?)(.+?)\\1\)/i';` (note the `*` instead of `+`, which would allow an empty url value) should also work.
Could you confirm whether that fixes your issue, and resubmit if so?
Thanks!

Reply:
an empty URL should not get parsed